docs/integrate/estuary/index.md

```diff
@@ -0,0 +1,117 @@
+(estuary)=
+
+# Estuary
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://estuary.dev/static/estuary-430cce9313e0df82d11e40d8285f38b4.png){w=180px}](https://estuary.dev/)
+```
```

Comment:
The logo is not too good with dark mode. @msbt: Please help!

![image](https://github.com/user-attachments/assets/e6ad50eb-e526-419e-95a6-a1e334ce500a)

/cc @michaelkremmel